MaxMind.MinFraud/Response/IIPAddress.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace MaxMind.MinFraud.Response
+{
+    /// <summary>
+    /// An interface for IP address risk classes.
+    /// </summary>
+    public interface IIPAddress
+    {
+        /// <summary>
+        /// The risk associated with the IP address. The value ranges from 0.01
+        /// to 99. A higher score indicates a higher risk.
```

Comment:
Is there a way to balance the doc tools and the json lib so that this summary appears only once? We have it in 3 different places in the source tree.